homeassistant/components/buienradar/sensor.py

```diff
@@ -227,8 +211,8 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     )
 
     dev = []
-    for sensor_type in config[CONF_MONITORED_CONDITIONS]:
-        dev.append(BrSensor(sensor_type, config.get(CONF_NAME), coordinates))
+    for sensor_type in SENSOR_TYPES:
+        dev.append(BrSensor(sensor_type, config[CONF_NAME], coordinates))
```

Comment:
I think we should disable these sensors by default, as I now end up with 107 entities.

![image](https://user-images.githubusercontent.com/195327/109531168-db31a600-7ab7-11eb-9485-c56f2dd49900.png)

Some might be helpful at some point for some people, but they can activate the needed sensors at that point.